examples/specs/layer_bar_annotations.vl.json

```diff
@@ -0,0 +1,60 @@
+{
+  "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
+  "description": "A simple bar chart with annotation.",
```

Comment:
I think this is more specific than just  "a simple bar chart with annotation".  

Do you have source of the original example?  It would be worth linking to giving the original example credits.

Reply:
I used ChartAccent tool to create the original example. Should I mention that where the dataset is from?